presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeSimpleQueries.java

```diff
@@ -129,7 +128,7 @@ public void testJsonFileBasedFunction()
      * The first query computes partial aggregation states and stores them in the avg_partial_states table.
      * Subsequent queries read from avg_partial_states and aggregate the states to the final result.
      */
-    @Test
+    @Test(enabled = false)
```

Comment:
Would you create a GitHub issue to provide some details about the test failure and add a link here?